main.css

```diff
@@ -372,14 +400,211 @@ body {
 
 #name:valid,
 #email:valid,
-#msg:valid {
+#msg:valid,
+#f-name:valid,
+#l-name:valid {
   box-shadow: 0 0 5px 1px hsl(155, 54%, 42%);
   border: 1px solid hsla(120, 100%, 25%, 0.506);
 }
 
 #name:focus:invalid,
 #email:focus:invalid,
+#f-name:focus:invalid,
+#l-name:focus:invalid,
 #msg:focus:invalid {
   box-shadow: 0 0 5px 1px hsl(0, 100%, 50%);
   border: 1px solid hsla(0, 100%, 50%, 0.48);
 }
+
+/* Desktop version CSS */
+
+@media screen and (min-width: 768px) {
+  body {
+    max-width: 1440px;
+    height: auto;
+    margin: 0 auto;
+  }
+
+  /* Display None functionality */
+
+  .header .navSec .logo,
+  .mobile-nav-btm {
+    display: none;
+  }
+
+  /* Header section Desktop */
+
+  .header {
+    position: absolute;
+    width: 100%;
+    height: 72px;
+    top: 0;
+    left: 0;
+  }
+
+  .header nav .navigation {
+    display: flex;
+    list-style: none;
+    align-items: center;
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓